apps/web/components/sections/admin/skeletons.tsx

```
@@ -0,0 +1,104 @@
+export function AdminOverviewSkeleton() {
+  return (
+    <SectionContainer className="py-8">
+      <div className="space-y-6">
+        <div className="space-y-2">
+          <SkeletonText className="h-8 w-48" />
+          <SkeletonText className="h-4 w-96" />
+        </div>
+        <div className="grid grid-cols-1 gap-4 md:grid-cols-2 lg:grid-cols-4">
+          {Array.from({ length: 4 }).map((_, i) => (
+            <div
+              key={i}
```

Biome (2.4.15)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)